styles.less

```diff
@@ -30,3 +30,7 @@ atom-text-editor {
 atom-text-editor::shadow .cursor {
   // border-color: red;
 }
+
+.tab .title:before {
```

Comment:
I have a patch open on the file-icons repo to support this config. We can hold off on merging until that gets updated.